common/build.gradle.kts

```diff
@@ -18,7 +17,7 @@ android {
     compileSdk = 33
```

Comment:
Should this also use the same compile SDK variable?

`libs.versions.androidSdk.compile.get().toInt()`